src/manifolds/Symplectic.jl

```diff
@@ -0,0 +1,889 @@
+@doc raw"""
+    Symplectic{n, ℝ} <: AbstractEmbeddedManifold{ℝ, DefaultIsometricEmbeddingType}
```

Comment:
```suggestion
    Symplectic{n, 𝔽} <: AbstractEmbeddedManifold{𝔽, DefaultIsometricEmbeddingType}
```
Corrected to match the definition

Reply:
:+1: